Source/WebCore/style/StyleBuilder.h

```diff
@@ -25,17 +25,20 @@
 
 #pragma once
 
-#include "CSSCustomPropertyValue.h"
 #include "PropertyCascade.h"
 #include "StyleBuilderState.h"
 #include <wtf/TZoneMalloc.h>
 
 namespace WebCore {
 
 struct CSSRegisteredCustomProperty;
+class CSSCustomPropertyValue;
+enum class CSSWideKeyword : uint8_t;
```

Comment:
I think we normally do these alphabetical, so `class` before `enum` before `struct`.